packages/credential-w3c/src/action-handler.ts

```diff
@@ -87,9 +87,19 @@ export class CredentialPlugin implements IAgentPlugin {
       createVerifiableCredential: this.createVerifiableCredential.bind(this),
       verifyCredential: this.verifyCredential.bind(this),
       verifyPresentation: this.verifyPresentation.bind(this),
+      createVerifiableCredentialOA:
+        this.createVerifiableCredentialOA.bind(this),
     };
   }
 
+  /** {@inheritdoc @veramo/core-types#ICredentialIssuer.createVerifiableCredential} */
+  async createVerifiableCredentialOA(
+    args: ICreateVerifiableCredentialArgs,
+    context: IssuerAgentContext
+  ): Promise<VerifiableCredential> {
+    throw new Error('Method not implemented.');
+  }
+
```

Comment:
The function is not used, should remove it?

Reply:
removed